bitcoin/src/blockdata/script/witness_version.rs

```diff
@@ -245,10 +245,14 @@ impl From<TryFromError> for TryFromInstructionError {
 
 /// Error attempting to create a [`WitnessVersion`] from an integer.
 #[derive(Clone, Debug, PartialEq, Eq)]
-#[non_exhaustive]
 pub struct TryFromError {
     /// The invalid non-witness version integer.
-    pub invalid: u8,
+    invalid: u8,
+}
+
+impl TryFromError {
+    /// Returns the invalid non-witness version integer.
+    pub fn invalid(&self) -> u8 { self.invalid }
```

Comment:
Yeah, this is too confusing I think. I'd prefer `provided_version` or something along those lines.

Reply:
Used `invalid_version`.